contents/docs/cdp/filter-out.md

```diff
@@ -80,6 +80,18 @@ You could use the following config:
   }
 ]
 ```
+Or, to filter out all traffic from a particular IP address, you could use a config like this example:
```

Comment:
```suggestion

Or, to filter out all traffic from a particular IP address, you can use a config like this:

```

Reply:
Ah, yes! Nice catch, thanks for removing that hedge. 😊